frontend/src/components/MailingForm.vue

```diff
@@ -97,8 +99,8 @@ export default {
   float:left;
 }
 
-.required
-{
+.required:after {
```

Comment:
Pretty sure we can remove this when we completely go off Vuetify, because using native html tags already have this input for us.

Reply:
yeah, for sure